src/scim2/scim2_endpoint.bal

```diff
@@ -0,0 +1,782 @@
+                    error e = error(SCIM2_ERROR_CODE
+                    , message = "Error occurred while accessing the payload of the response.");
+                    return e;
+                }
+            }
+        } else {
+            error err = error(SCIM2_ERROR_CODE, message = "Error occurred while invoking the SCIM2 API");
+            return err;
+        }
+    }
+
+    # Returns whether the user specified by username belongs to the group specified by groupname.
+    # + userName - User name of the user
+    # + groupName - Name of the group
+    # + return - If success, returns boolean value, else returns error
+    public remote function isUserInGroup(string userName, string groupName) returns (boolean | error) {
+        http:Request request = new ();
+        User user = {};
+
+        var res = self.scimClient->get(SCIM_USER_END_POINT + "?" + SCIM_FILTER_USER_BY_USERNAME + userName
+        , message = request);
+        if (res is http:Response) {
+            var receivedUser = resolveUser(userName, res);
+            if (receivedUser is User) {
+                user = receivedUser;
+                foreach var gro in user.groups {
+                    if (stringutils:equalsIgnoreCase(gro.displayName, groupName)) {
+                        return true;
+                    }
+                }
+                return false;
+            } else {
+                error err = error(SCIM2_ERROR_CODE, message = "Unable to get the given userName :" + userName);
+                return err;
+            }
+        } else {
+            error err = error(SCIM2_ERROR_CODE, message = "Error occurred while invoking the SCIM2 API");
+            return err;
+        }
+    }
+
+    # Delete a user from user store.
+    # + userName - User name of the user
+    # + return - If success, returns string message with status, else returns error
+    public remote function deleteUserByUsername(string userName) returns (string | error) {
+        http:Request request = new ();
+
+        string failedMessage;
+        failedMessage = "Deleting user:" + userName + " failed. ";
+
+        //get user
+        http:Request userRequest = new ();
+        User user = {};
+        var resUser = self.scimClient->get(SCIM_USER_END_POINT + "?" + SCIM_FILTER_USER_BY_USERNAME + userName,
+        message = userRequest);
+        if (resUser is http:Response) {
+            var receivedUser = resolveUser(userName, resUser);
+            if (receivedUser is User) {
+                user = receivedUser;
+                string userId = user.id;
+                var res = self.scimClient->delete(SCIM_USER_END_POINT + SCIM_FILE_SEPERATOR + userId, request);
+                if (res is http:Response) {
+                    if (res.statusCode == HTTP_NO_CONTENT) {
+                        return "deleted";
+                    } else {
+                        error Error = error(SCIM2_ERROR_CODE, message = failedMessage + res.reasonPhrase);
```

Comment:
Capital variable name _Error_ is used. B7A variable names are in Camel Case in convention.

Reply:
Fixed